packages/voice/examples/basic/adapter.ts

```diff
@@ -1,5 +1,5 @@
 import { DiscordGatewayAdapterCreator, DiscordGatewayAdapterLibraryMethods } from '../../';
-import { Snowflake, Client, Guild, VoiceBasedChannel, Status, Events } from 'discord.js';
+import { Snowflake, Client, Guild, VoiceBasedChannel, Status, Events } from 'discord';
```

Comment:
```suggestion
import { Snowflake, Client, Guild, VoiceBasedChannel, Status, Events } from 'discord.js';
```

wrong replace (same in other example files too)